src/main/java/com/github/idea/ginkgo/util/GinkgoUtil.java

```diff
@@ -56,7 +56,10 @@ private static Optional<String> getSpecDescription(GoCallExpr ginkgoSpecFunction
     }
 
     public static String escapeRegexCharacters(String specName) {
-        return specName.replace("\"", "").replace("(", "\\(").replace(")", "\\)");
+        return specName.replace("\"", "")
+                .replace("(", "\\(")
+                .replace(")", "\\)")
+                .replace("|", "\\\\|");
```

Comment:
Can you update the tests to capture this [here](https://github.com/IdeaGinkgo/Intellij-Ginkgo/blob/master/src/test/java/com/github/idea/ginkgo/util/GinkgoUtilTest.java#L34)

Reply:
Updated, test pass successfully
![image](https://user-images.githubusercontent.com/18268219/229374383-5ad6a11f-2102-4651-ab13-ba2a8479c6d9.png)